features/features.go

```diff
@@ -224,6 +224,13 @@ var (
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
+	FeatureGateBGP = newFeatureGate("BGP").
```

Comment:
Might want a slightly more specific name for this gate, what exactly with BGP are we doing here